<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VPBTerrain</class>
 <widget class="QWidget" name="VPBTerrain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>326</width>
    <height>214</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VPB生成地形</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icon.qrc">
    <normaloff>:/icon/32x32/构建.png</normaloff>:/icon/32x32/构建.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>LOD等级         </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lEdt"/>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>垂直比率        </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="vEdt"/>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>设置坐标系统    </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="csEdt"/>
       </item>
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>纹理文件        </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="tEdt"/>
       </item>
       <item>
        <widget class="QPushButton" name="tBtn">
         <property name="text">
          <string>选择文件</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>DEM文件         </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="dEdt"/>
       </item>
       <item>
        <widget class="QPushButton" name="dBtn">
         <property name="text">
          <string>选择文件</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>数据页输出文件  </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="oEdt"/>
       </item>
       <item>
        <widget class="QPushButton" name="oBtn">
         <property name="text">
          <string>设置文件</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QPushButton" name="okBtn">
         <property name="text">
          <string>确定</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="closeBtn">
         <property name="text">
          <string>关闭</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="helpBtn">
         <property name="text">
          <string>帮助</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
